src/util/closeNonClosedHtmlTags.ts

```diff
@@ -0,0 +1,8 @@
+import truncate from 'truncate-html'
+
+/** Close non closed html tags. */
+const closeNonClosedHtmlTags = (html :string) => {
+  return truncate(html, html.length)
+}
+
+export default closeNonClosedHtmlTags
```

Comment:
Since `closeNonClosedHtmlTags` is only used in one file, it would be better to define it internal to that module in order to avoid increasing the surface area of the util api.

The separate file does decouple `convertHMTLtoJSON` from the `truncate-html` dependency, but it's not a major interface with large implementation details to hide, so I think the simpler architectural is a better fit.